src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(firstValue) {
+    return secondValue => {
```

Comment:
it'd be better to use `()` always, so it's more clear where parameters start and end
```suggestion
    return (secondValue) => {
```